apps/ffdweb-site/src/app/_constants/featuredProjects.ts

```diff
@@ -0,0 +1,14 @@
+export const FEATURED_PROJECTS = [
```

Comment:
Does this need to be an object? Can it be an array of strings?

Reply:
Good point! I can move in the `(homepage)` folder since we only use it there for now.